packages/tldraw/src/lib/ui/getLocalFiles.ts

```diff
@@ -9,17 +9,22 @@ export function getLocalFiles(options?: {
 		input.type = 'file'
 		input.accept = mimeTypes?.join(',')
 		input.multiple = allowMultiple
+		input.style.display = 'none'
 
 		async function onchange(e: Event) {
 			const fileList = (e.target as HTMLInputElement).files
-			if (!fileList || fileList.length === 0) return
+			if (!fileList || fileList.length === 0) {
+				resolve([])
+				return
+			}
 			const files = Array.from(fileList)
 			input.value = ''
 			resolve(files)
 			input.removeEventListener('change', onchange)
 			input.remove()
 		}
 
+		document.body.appendChild(input)
 		input.addEventListener('change', onchange)
```

Comment:
you'll also need to listen to the `cancel` event so that we clean up things properly.
(prbly have a `dispose` function that we call in both cases to clean up stuff